drivers/project_scorecard/app/views/project_scorecard/warehouse_reports/scorecards/_grant_management_and_financials.haml

```diff
@@ -14,7 +14,7 @@
       %th
         Projected Project Spend Down
       %th
-        = simple_format(_('Ensure the program is on track to spend down all project funds by the end of its current grant term'))
+        = simple_format(_('Ensure the program is on track to spend down all project funds by the end of its current grant term Agency Self-Report and eLOCCS data. For sub-contracted programs, subtract one month for the "number of months since start of grant term" to allow for the HUD billing delay in sub-contracted programs.'))
```

Comment:
Should there be a period before "Agency Self-Report"?

Reply:
Looks like that's actually a duplicate of line 19.  Removing.